addons/account/models/account_tax.py

```diff
@@ -2325,6 +2325,9 @@ def _exclude_tax_groups_from_tax_totals_summary(self, tax_totals, ids_to_exclude
     # TAX LINES GENERATION
     # -------------------------------------------------------------------------
 
+    def _should_keep_zero_lines(self, record):
```

Comment:
```suggestion
    def _should_keep_zero_tax_lines(self, tax_line):
```
To stay consistent with the others usage of `_prepare_tax_lines` (bank rec widget, expenses, etc), what about relying on the tax_line directly and make it agnostic of the business record?